submissions/Nik3264/Building-a-Tiny-JS-World/index.js

```diff
@@ -0,0 +1,79 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _https://github.com/Nik3264/a-tiny-JS-world_
+   Web app: _https://nik3264.github.io/a-tiny-JS-world/_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Toby',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'woof-woof!'
+ };
+
+ const cat = {
+   species: 'cat',
+   name: 'Vasya',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'mi-mi!'
+ };
+ const catwoman = {
+   species: 'catwoman',
+   name: 'Toby',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: cat.saying
+ };
+ const men = {
+   species: 'men',
+   name: 'Toby',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'hello!'
+ };
+ const woman = {
+   species: 'woman',
+   name: 'Tyna',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'wo hei!'
+ };
+
+ const arr = [dog, cat, catwoman, men, woman];
+
+ const str = arr.map(el=>{
```

Comment:
Don't forget to add spaces around ` => `, ` = ` and pretty much everything. It helps to mare the code clearer.

Reply:
Thank you, I correct it.